hera_mc/cm_active.py

```diff
@@ -73,9 +73,10 @@ def load_parts(self, at_date=None):
         at_date = cm_utils.get_astropytime(at_date)
         gps_time = self.set_times(at_date)
         self.parts = {}
-        for prt in self.session.query(partconn.Parts).filter((partconn.Parts.start_gpstime <= gps_time)
-                                                             & ((partconn.Parts.stop_gpstime >= gps_time)
-                                                             | (partconn.Parts.stop_gpstime == None))):  # noqa
+        for prt in self.session.query(partconn.Parts).filter(
+            (partconn.Parts.start_gpstime <= gps_time)
+            & ((partconn.Parts.stop_gpstime >= gps_time)
+               | (partconn.Parts.stop_gpstime == None))):  # noqa
```

Comment:
This must be due to the max line-length change, but I personally find it very difficult to discern where `for` statement ends. Could we move the `):` to a new line perhaps?

Reply:
sure. I will probably get used to that with black. Still is hard for me to read, but we're going that way it seems.